.storybook/preview.tsx

```diff
@@ -86,6 +86,13 @@ export default {
     docs: {
       codePanel: true,
     },
+
+    a11y: {
+      // 'todo' - show a11y violations in the test UI only
```

Comment:
Does "test UI" mean Storybook web UI?

```suggestion
      // 'todo' - show a11y violations in the web UI only
```

Reply:
No, i think it meant the small widget in bottom-left corner when running storybook. This comment was auto-generated when setting up test addon.